rxnetty-http/src/main/java/io/reactivex/netty/protocol/http/client/internal/RawRequest.java

```diff
@@ -166,7 +175,7 @@ public Observable asObservable(Connection<?, ?> connection) {
         return toReturn;
     }
 
-    private HttpRequest _copyHeaders() {
+    public HttpRequest _copyHeaders() {
```

Comment:
huh, that's correct, I didn't notice the other method is public, shouldn't be. Do you mind making that private, since you are on it :)